docs/source/appendix.rst

```diff
@@ -1,8 +1,62 @@
-Zipline API
------------
+API Reference
+-------------
 
-For each of these api functions, the ``self`` argument is implicitly the current
-:class:`~zipline.algorithm.TradingAlgorithm`
+Algorithm API
+~~~~~~~~~~~~~
+
+The following methods are available for use in the ``initialize``,
+``handle_data`` and ``before_trading_start`` API functions.
```

Comment:
(That said, I generally fall on the pro-comma side of said debates, so I'll add one.)